docs/workflows/flatpak.rst

```diff
@@ -0,0 +1,64 @@
+.. _flatpak-workflow:
+
+Hosting Flatpak Content in OCI Format
+=====================================
+
+Pulp can host Flatpak application and runtime images that are distributed in OCI format.  To make
```

Comment:
```suggestion
Pulp can host Flatpak applications and runtime images that are distributed in OCI format.  To make
```

Reply:
that should be fine, short for "application images and runtime images"